CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ The changes are relative to the previous release, unless the baseline is specifi
 * Write an empty HandlerBox name field instead of "libavif" (saves 7 bytes).
 * Update aom.cmd/LocalAom.cmake: v3.10.0
 * Update avm.cmd: research-v8.0.0
+* Update dav1d.cmd/LocalDav1d.cmake/dav1d_android.sh: 1.5.0
```

Comment:
Nit: List dav1d_android.sh before LocalDav1d.cmake, because dav1d.cmd and dav1d_android.sh are in the same directory.